### DISCUSSION TITLE
gestion de inventario

### DISCUSSION THREAD

**Discussion by lan191** [gdarko/digital-license-manager#22] [Ideas]
hola, felicitaciones por el magnifico programa. tengo una sugerencia me gustaria que se pueda controlar la cantidad de inventario cuando se agrega pocas licencias. 
![gestion 28-02-2023](https://user-images.githubusercontent.com/88171015/221998542-5a1a61b3-4b6e-49ee-b84a-a42e699306d6.png)
![gestion1 28-02-2023](https://user-images.githubusercontent.com/88171015/221998561-8b1e8ce8-45ff-4684-b97a-18aef46244b7.png)

**Comment by lan191** [comment]
el inventario lo controlo manualmente pero me gustaria que sea automatico, asi el cliente podrá elegir la cantidad de licencias de acuerdo al inventario. cuando el cliente agrega más licencias de los que hay en el inventario se crea un error.

**Comment by gdarko** [comment]
This issue has been fixed in our previous releases. Thanks.
